apps/full-site-editing/full-site-editing-plugin/global-styles/src/no-support.js

```diff
@@ -0,0 +1,3 @@
+export default ( { unsupportedFeature } ) => (
+	<p>{ `Your current theme doesn't support ${ unsupportedFeature } customization.` }</p>
```

Comment:
Should you add `__()` and `sprintf` here?

Reply:
fixed at https://github.com/Automattic/wp-calypso/pull/36288/commits/abb67d1c0f118679c34626dcb0876c7de91eb344